src/templates/json-editor.html

```diff
@@ -11,9 +11,9 @@ <h6 id='jsonEditorTitle'></h6>
     {% if mode == 'edit' %}
       <div class='mt-4 text-center'>
         <div class='btn-group'>
-          <button type='button' data-action='json-save' class='btn btn-primary'>{{ 'Save'|trans }}</button>
+          <button type='button' id='json-save' data-action='json-save' class='btn btn-primary'>{{ 'Save'|trans }}</button>
```

Comment:
is there a reason for adding an id here?

Reply:
The ID is used to address the html element in the JS code. I think using ID attributes is a better solution because `getElementById()` is more efficient than the query selector. Of course this cannot be applied everywhere.